### DISCUSSION TITLE
Markdown tree-sitter tags extension

### DISCUSSION THREAD

**Discussion by sicher** [helix-editor/helix#8700] [General]
There is an extension in the Markdown tree-sitter for #tags... Given that the just released marksman lsp is including support for looking up tag references it would be great if helix would include tree-sitter support for highlighting them.

If this is just a build config change I can probably do it myself, if given a pointer where to do it.

**Comment by Idobenhamo** [comment]
How the search for tag references with helix is used?

**Reply by sicher** [reply]
When on a tag, press `gr` to list all references.

**Comment by the-mikedavis** [comment]
https://github.com/helix-editor/helix/pull/8749 has an example solution for this when using the flake. The optional features in tree-sitter-markdown are gated by environment variables being set when running `tree-sitter generate`. We don't run that ourselves: we compile the C files that `tree-sitter generate` creates (`src/{parser.c,scanner.*}`), so this could either be solved while building the grammar if you have control over that (like in the flake) or you could fork/clone tree-sitter-markdown and generate the parser with those extensions enabled.

**Comment by mzfr** [comment]
Hi, I apologize for commenting on an old thread, but has this been added yet? Is there a direct or simpler way to highlight `#tags` without having to build the grammar?
